content/docs/dvclive/api-reference/live/index.md

```diff
@@ -21,59 +21,71 @@ from dvclive import Live
 live = Live()
 ```
 
+Or use as a context manager:
+
+```py
+from dvclive import Live
+
+with Live() as live:
+    ...
+```
+
```

Comment:
Non-blocking: let's only keep the context manager (see https://github.com/iterative/dvc.org/pull/4085/files#r1014164477).